bundles/model/org.eclipse.smarthome.model.item.ui/src/org/eclipse/smarthome/model/ui/contentassist/ItemsProposalProvider.xtend

```diff
@@ -1,3 +1,10 @@
+/**
+ * Copyright (c) 2014-2016 by the respective copyright holders.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
 /*
  * generated by Xtext
```

Comment:
As these files are generated as the Javadoc says, they are "inferred" files, i.e. imho not really be treated as source code (because nobody wrote it). As they also get overwritten on a potential re-generation, I would refrain from adding a license header to them.